meson.build

```diff
@@ -1,7 +1,9 @@
 project(
     'vs-fltk',
     ['c', 'cpp', 'swift'],
-    default_options: ['c_std=gnu23', 'cpp_std=gnu++23'],
+    version: '0.1.0-dev',
+    meson_version: '>= 1.1',
```

Comment:
Build type: native build
WARNING: Project does not target a minimum version but uses feature introduced in '1.1': meson.options file. Use meson_options.txt instead